cdap-ui/app/features/workflows/templates/detail.html

```diff
@@ -1,39 +1,199 @@
 <my-breadcrumbs></my-breadcrumbs>
-<div class="row" program-heading>
-  <div class="col-sm-6">
-    <h2 class="programTitle">
-      <span ng-bind="$state.params.programId"></span>
-    </h2>
-    <div class="type-block">
-      <span class="icon-workflow"></span>
-      <small> Type: WorkFlow </small>
+
+<div class="levels">
+  <div class="program-level">
+    <div class="container">
+      <div class="row" program-heading>
+        <div class="col-sm-4 col-md-6">
+          <h2 ng-bind="$state.params.programId"></h2>
+          <div class="type-block">
+            <span class="icon-workflow"></span>
+            <small> Type: Workflow </small>
+          </div>
+          <p> Description goes here. </p>
```

Comment:
May be we should remove this? We need to decide on how we are going to show description on the detail view.